src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let numberCall = 0;
+
+  return function(x) {
+    return function(y) {
+      numberCall++;
+
+      if (numberCall > 3 && numberCall % 2 === 0) {
+        return 'Bzzz... Error!';
+      }
+
+      return x + y;
+    };
+  };// write code here
```

Comment:
```suggestion
  };
```
Don't forget to delete comments